streaming/index.js

```diff
@@ -133,13 +133,26 @@ const redisConfigFromEnv = (env) => {
   // which means we can't use it. But this is something that should be looked into.
   const redisPrefix = env.REDIS_NAMESPACE ? `${env.REDIS_NAMESPACE}:` : '';
 
-  const redisParams = {
+  let redisParams = {
     host: env.REDIS_HOST || '127.0.0.1',
     port: env.REDIS_PORT || 6379,
     db: env.REDIS_DB || 0,
     password: env.REDIS_PASSWORD || undefined,
   };
 
+  if (env.REDIS_SENTINEL) {
+	redisParams = {
+	  password: env.REDIS_PASSWORD || undefined,
+	  sentinels: env.REDIS_SENTINEL.split(',').map(h => {
+		const s = h.split(':');
+		const host = s[0] || '127.0.0.1';
+		const port =  s[1] || '26379';
+		return { "host": host, "port": port };
+	  }),
```

Comment:
This also doesn't do the same DNS resolving logic that ruby does.

Reply:
I believe ioredis does this for us, but will have to double check.